batch/batch/worker/worker.py

```diff
@@ -567,30 +765,68 @@ async def delete(self):
     #   timing: dict(str, float),
     #   error: str, (optional)
     #   short_error: str, (optional)
-    #   container_status: { (from docker container state)
+    #   container_status: {
     #     state: str,
     #     started_at: str, (date)
     #     finished_at: str, (date)
-    #     out_of_memory: bool
-    #     error: str, (one of error, exit_code will be present)
```

Comment:
also verify that error is not used anywhere

Reply:
This whole `container_status['container_status']` and repeated fields are quite confusing imo. I think it was only used in one place in the batch client so I removed it (but left in the out_of_memory), and will try to reconcile the tests. Since the inner `container_status` was essentially passing through the docker status I would like to follow up by flattening the container status structure. I don't see a scenario where the duplicate fields have/should have different values.